.github/workflows/build.yml

```diff
@@ -550,6 +602,7 @@ jobs:
     - build_ubuntu_ssltests
     - test_hypothesis
     - build_asan
+    - cifuzz
```

Comment:
Please could you also add this to the `allowed-failures:` and  `allowed-skips:` sections?

Reply:
Done ✅